WindowManager/app/src/main/java/com/example/windowmanagersample/WindowDemosActivity.kt

```diff
@@ -40,17 +42,15 @@ class WindowDemosActivity : AppCompatActivity() {
         val view = binding.root
         setContentView(view)
 
-        binding.backendRadioGroup.setOnCheckedChangeListener { _, checkedId ->
-            when (checkedId) {
-                R.id.device_default_radio_button ->
-                    selectedBackend = BACKEND_TYPE_DEVICE_DEFAULT
-                R.id.mid_fold_radio_button ->
-                    selectedBackend = BACKEND_TYPE_MID_SCREEN_FOLD
+        val radioGroup = binding.backendRadioGroup
+        radioGroup.setOnCheckedChangeListener { _, checkedId ->
+            selectedBackend = when (checkedId) {
+                R.id.deviceDefaultRadioButton -> BACKEND_TYPE_DEVICE_DEFAULT
+                R.id.shortDimensionFoldRadioButton -> BACKEND_TYPE_SHORT_DIMENSION_FOLD
+                R.id.longDimensionFoldRadioButton -> BACKEND_TYPE_LONG_DIMENSION_FOLD
+                else -> BACKEND_TYPE_DEVICE_DEFAULT
             }
         }
-        binding.featuresActivityButton.setOnClickListener { showDisplayFeatures() }
-
-        binding.splitLayoutActivityButton.setOnClickListener { showSplitLayout() }
```

Comment:
do we feel like the xml onClick properties are more readable to cleaner than setting the listeners here?

Reply:
I've reverted to setting the listeners in the onCreate.